sdc/rabbit/publisher.py

```diff
@@ -116,16 +111,19 @@ def publish_message(self, message, content_type=None, headers=None, mandatory=Fa
 
             logger.info('Published message to queue queue={}'.format(self._queue))
             return result
+        except pika.exceptions.AMQPConnectionError:
+            logger.error("Message not published. Published Message raised")
```

Comment:
Better to say _PublishMessageError_ raised? Sentence won't read clearly without context in future logs. Or better say something like "Message not published due to connection error" or similar.